src/bdc/steps/social_media_api.py

```diff
@@ -14,6 +14,54 @@
 log = get_logger()
 
 
+def is_company_email(email):

```

Comment:
Is this logic not already done by the analyze emails step?

Reply:
Indeed, Independent that was more straight forward to me. However I established the dependency now. Please check it